### PR TITLE
tip preventing gradle error/hanging on windows

### PR DESCRIPTION
**Pull request by kaidohallik**
discussed in jhipster/generator-jhipster#4809

### CODE REVIEW

File: tips/021_tip_prevent_error_and_hanging_on_windows_when_using_gradle.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+layout: default
```
Comment (jdubois) [inline tips/021_tip_prevent_error_and_hanging_on_windows_when_using_gradle.md]: Can you check your carriage returns?

File: tips/021_tip_prevent_error_and_hanging_on_windows_when_using_gradle.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
+---
+layout: default
+title: Prevent error/hanging on Windows when using gradle
+sitemap:
+priority: 0.5
+lastmod: 2017-01-03T23:00:00-00:00
+---
+
+# Prevent error/hanging on Windows when using gradle
```
Comment (jdubois) [inline tips/021_tip_prevent_error_and_hanging_on_windows_when_using_gradle.md]: gradle -> Gradle

File: tips/021_tip_prevent_error_and_hanging_on_windows_when_using_gradle.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+* `gradlew --info`
+* `gradlew --debug`
+* `gradlew --console plain`
+
+The problem can sometimes be prevented with the following actions:
+
+* increase the cmd.exe window Screen Buffer Size Height, for example to the maximum possible value 9999
+* open new cmd.exe window and use this for executing gradle tasks
+
+In the jhipster applications this problem has reproduced with the following gradle tasks:
```
Comment (jdubois) [inline tips/021_tip_prevent_error_and_hanging_on_windows_when_using_gradle.md]: jhipster -> JHipster

Comment (jdubois) [inline tips/021_tip_prevent_error_and_hanging_on_windows_when_using_gradle.md]: gradle -> Gradle

File: tips/021_tip_prevent_error_and_hanging_on_windows_when_using_gradle.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+* gulpConstantDev
+* gulpBuildWithOpts
+* npmInstall
+* yarn_install
+* webpack
+
+Exists hack which prevents this error/hanging in the jhipster applications. Just put the following code to your `build.gradle`:
```
Comment (jdubois) [inline tips/021_tip_prevent_error_and_hanging_on_windows_when_using_gradle.md]: jhipster -> JHipster

File: tips/021_tip_prevent_error_and_hanging_on_windows_when_using_gradle.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+tasks.withType(com.moowork.gradle.node.yarn.YarnTask) {
+    doLast {
+        println()
+    }
+}
+```
+
+## Additional information
+
+Exists gradle issue about that problem and currently work in progress on that issue.
```
Comment (jdubois) [inline tips/021_tip_prevent_error_and_hanging_on_windows_when_using_gradle.md]: gradle -> Gradle

File: tips/021_tip_prevent_error_and_hanging_on_windows_when_using_gradle.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+It's possible that in the future versions of gradle the problem doesn't exist any more.
```
Comment (jdubois) [inline tips/021_tip_prevent_error_and_hanging_on_windows_when_using_gradle.md]: gradle -> Gradle

File: tips/021_tip_prevent_error_and_hanging_on_windows_when_using_gradle.md (hunk 5 above)
Comment (jdubois) [inline tips/021_tip_prevent_error_and_hanging_on_windows_when_using_gradle.md]: Wording: there is no subject in your sentence

**Comment by kaidohallik** [comment]
@jdubois I made suggested changes and updated PR.

**Comment by jdubois** [comment]
Thanks! I'm merging this.

**Pull request merged**